libc/src/__support/threads/linux/CMakeLists.txt

```diff
@@ -64,6 +64,7 @@ add_object_library(
     .futex_utils
     libc.config.linux.app_h
     libc.include.sys_syscall
+    libc.include.fcntl
```

Comment:
will changing `thread.cpp` to use `hdr/fcntl_macros.h` and add `libc.hdr.fcntl_macros` dependency work?

Reply:
I suppose thread.cpp is built only in full build mode?

Reply:
You're right: https://github.com/llvm/llvm-project/issues/93241